src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+@import "../blocks/variebles.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
+  padding: 33px 16px 16px;
+
+  &__img {
+    margin-bottom: 40px;
+    max-width: 100%;
+  }
+
+  &__product {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &:hover &__product {
+    color: $product-hover;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    font-size: 12px;
+    font-weight: 400;
```

Comment:
Still not fixed, find all similar lines in the code and fix them all
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/50254f66-1478-4e6a-9556-fc205e4e9288)